packages/js/src/components/modals/editorModals/FacebookPreviewModal.js

```diff
@@ -4,6 +4,17 @@ import { __ } from "@wordpress/i18n";
 /* Internal dependencies */
 import EditorModal from "../../../containers/EditorModal";
 import FacebookEditor from "../../../containers/FacebookEditor";
+import React from "react";
```

Comment:
Importing `React` in this file is restricted and this causes an error in the linting check. The import statement should be removed